src/igraph/io/libraries.py

```diff
@@ -11,7 +11,7 @@ def _export_graph_to_networkx(
         constructing the graph. C{None} means to let igraph infer the most
         appropriate class based on whether the graph is directed and whether
         it has multi-edges.
-    @param vertex_attr_hashable (str): vertex attribute used to name vertices
+    @param vertex_attr_hashable: (str) vertex attribute used to name vertices
```

Comment:
You can remove `(str)` from here. It used to be a type annotation, but in the future we will migrate to "proper" type annotations in the function headers so there's no use in keeping it there. (Plus, it is not recognized as a type annotation by the parser anyway; I think this style of annotation works only in docstrings that follow the Google style).

Reply:
yes